### PR TITLE
Print ps v <pid> debug message for linux and aix in LoaderSlaveMultiThread

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
- Print ps v <pid> debug message for linux and aix in LoaderSlaveMultiThread
- Related to https://github.com/eclipse/openj9/issues/8842

Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Test: 
Windows: Test_grinder/job/Grinder/6791/console
Linux: Test_grinder/job/Grinder/6790/
Aix: Test_grinder/job/Grinder/6789/console

**Comment by Mesbah-Alam** [comment]
@pshipton  - please review.

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/LoaderSlaveMultiThread.java [hunk 1]
```diff
@@ -55,13 +57,42 @@ public static void main(String args[])
 		try
 		{
 			(new LoaderSlaveMultiThread()).run(args[0],Integer.parseInt(args[1]));
+			
+			// Print debug information: on Linux or AIX, run ps v <pid> on the JVM to get the RSS from the SZ column
```
Comment (pshipton) [inline openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/LoaderSlaveMultiThread.java]: Pls remove "from the SZ column".  Using the new open `v` there is no SZ column, but a RSS column.

**Comment by pshipton** [comment]
Approved assuming the comment change is made.

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/LoaderSlaveMultiThread.java (hunk 1 above)
Comment (Mesbah-Alam) [reply openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/LoaderSlaveMultiThread.java]: I've updated the comment. Thanks!

Comment (Mesbah-Alam) [reply openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/LoaderSlaveMultiThread.java]: Please merge it.

**Pull request merged**